### DISCUSSION TITLE
Disable incoming mails

### DISCUSSION THREAD

**Discussion by devopsmash** [postalserver/postal#2020] [Help with using Postal] [answered]
I'm using Postal for Outgoing mails, and for security reason, I'm wondering if there is any way to disable Incoming mails?

note: I need to keep port 25 open, because my app is using this port and SMTP credentials to send mails.

**Comment by willpower232** [comment, marked as answer]
Incoming mails need to work to allow return path messages to be received by Postal.

**Reply by devopsmash** [reply]
Thank you @willpower232 for your reply.
And if I can live without the return path (I know that I won't get any bounced mails) is it possible to disable the processing of incoming mails somehow?

**Reply by willpower232** [reply]
I don't believe so no. You could limit access to the port via firewall however it is also possible that systems that receive your email messages will verify that where they received the message from is capable of receiving replies so it might cause you more problems than it solves.
